roles/os_update_latest/tasks/main.yml

```diff
@@ -1,3 +1,10 @@
 ---
+- name: Make room for new kernel on /boot
+  command: /usr/bin/package-cleanup --oldkernels --count=2 -y
```

Comment:
This seems to be solving a very specific environmental problem. If the boot partition isn't large enough to accommodate the default number of installed kernels that's either a bug in the OS because the kernel has become too bloated or the provisioning process which has created a /boot filesystem that's unreasonably small.

Reply:
The problem really is that old initramfs images get left in /boot.  Even old ones from 3, 4, 5... versions ago where the kernel has been removed.  This just ensures that those are removed.